Added new Playbook - Push Defender for Endpoint Alerts including Alert Names, MITRE Tactics, Techniques and Sub-Techniques as Hunting ARM Templates into a Sentinel Github Repository

SCStelz opened the pull request:
Required items, please complete
   
   Change(s):
   - Added new Playbook - Push Defender for Endpoint Alerts including Alert Names, MITRE Tactics, Techniques and Sub-Techniques as Hunting ARM Templates into a Sentinel Github Repository

   Reason for Change(s):
   - New Playbook Added

   Testing Completed:
   - Yes, fully tested

v-spadarthi commented:
@manishkumar1991 / @rahul0216 : Please have a look and provide your feedback. Thanks!

v-spadarthi commented:
@manishkumar1991 / @rahul0216 : Please have a look and provide your feedback. Thanks!

v-spadarthi commented:
@manishkumar1991 / @rahul0216 : Please have a look and provide your feedback. Thanks!

v-mchatla commented:
Hi @rahul0216,
Can you please review the playbook and share your thoughts.
Thanks

v-spadarthi commented:
@manishkumar1991 / @rahul0216 : Please have a look and provide your feedback. Thanks!

v-spadarthi commented:
@manishkumar1991 / @rahul0216 : Please have a look and provide your feedback. Thanks!

manishkumar1991 commented:
We are reviewing it , please allow sometime

v-spadarthi commented:
@manishkumar1991 / @rahul0216 : Please have a look and provide your feedback. Thanks!

manishkumar1991 commented:
@SCStelz , kindly use the playbook arm template generator tool for creating the playbook template and then fill out all the necessary meta data ,

Kindly check below link for arm template generator
https://github.com/Azure/Azure-Sentinel/tree/master/Tools/Playbook-ARM-Template-Generator

kindly check below link for reference on metadata details 
https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/ThreatXCloud/Playbooks/ThreatXPlaybooks/ThreatX-BlockIP-URL

v-spadarthi commented:
@SCStelz : Please address the @manishkumar1991 comments. Thanks!

v-spadarthi commented:
@SCStelz : Please address the @manishkumar1991 comments. Thanks!

SCStelz commented:
I did use the template Generator, wasn't aware of the meta data, azuredeploy.json have been updated /w the require metadata.

From: Manish Kumar ***@***.***>
Sent: Wednesday, December 28, 2022 4:36 AM
To: Azure/Azure-Sentinel ***@***.***>
Cc: Chris Stelzer ***@***.***>; Mention ***@***.***>
Subject: Re: [Azure/Azure-Sentinel] Added new Playbook - Push Defender for Endpoint Alerts including Alert Names, MITRE Tactics, Techniques and Sub-Techniques as Hunting ARM Templates into a Sentinel Github Repository (PR #6803)


@SCStelz<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FSCStelz&data=05%7C01%7Cchris.stelzer%40microsoft.com%7Ccbfb17a99f2341f3247c08dae8d024ec%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638078277924951268%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=wXDVsnyDAQuh4SFIl0j3omyNErR5r0MgZajBSrfTkzQ%3D&reserved=0> , kindly use the playbook arm template generator tool for creating the playbook template and then fill out all the necessary meta data ,

Kindly check below link for arm template generator
https://github.com/Azure/Azure-Sentinel/tree/master/Tools/Playbook-ARM-Template-Generator<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2FAzure-Sentinel%2Ftree%2Fmaster%2FTools%2FPlaybook-ARM-Template-Generator&data=05%7C01%7Cchris.stelzer%40microsoft.com%7Ccbfb17a99f2341f3247c08dae8d024ec%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638078277924951268%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=kblW7vLtO0awJS%2BQb%2Fx40OG%2Bl9HLEz1FE3Ddq8I1pNo%3D&reserved=0>

kindly check below link for reference on metadata details
https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/ThreatXCloud/Playbooks/ThreatXPlaybooks/ThreatX-BlockIP-URL<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2FAzure-Sentinel%2Ftree%2Fmaster%2FSolutions%2FThreatXCloud%2FPlaybooks%2FThreatXPlaybooks%2FThreatX-BlockIP-URL&data=05%7C01%7Cchris.stelzer%40microsoft.com%7Ccbfb17a99f2341f3247c08dae8d024ec%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638078277924951268%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Tp2zT%2FYqxwUbRK8FwkPx2BezW1YfFqanMg1fI2unQME%3D&reserved=0>

-
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2FAzure-Sentinel%2Fpull%2F6803%23issuecomment-1366620532&data=05%7C01%7Cchris.stelzer%40microsoft.com%7Ccbfb17a99f2341f3247c08dae8d024ec%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638078277924951268%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Mu8T%2BM9QmjDny3k%2Bn2hhkt%2FbsVV36sga0bSi8PsTiuQ%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FARQXT66MCOOWTH347D7KNMTWPQX43ANCNFSM6AAAAAASSMAXPE&data=05%7C01%7Cchris.stelzer%40microsoft.com%7Ccbfb17a99f2341f3247c08dae8d024ec%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638078277924951268%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=6tNBS2Ql3PjzZG2aa7f6tCcPw7FTsS8AXP4Nqm%2BegEM%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

v-spadarthi commented:
@manishkumar1991 : Please have a look once again your comments addressed by @SCStelz .

manishkumar1991 commented:
> @manishkumar1991 : Please have a look once again your comments addressed by @SCStelz .

will check today

manishkumar1991 commented on Playbooks/Put-MDEAlert-Hunting-GitHub/readme.md:
```diff
@@ -0,0 +1,78 @@
+
+You will need to generate a [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for your Github Repository associated with Sentinel with the Repo, and Workflow permissions.
+
+You will need your Personal Access Token from GitHub stored in Keyvault and have the URL of the secret handy.
```
@SCStelz , By URL of secret, do you mean by the URL present inside the secret like mentioned in screen shot below .
![image](https://user-images.githubusercontent.com/97503740/211983484-1a448e14-7716-4e0b-b303-309215d6182a.png)

if Yes , then please mention the steps as well on how to fetch the url of the secret. so that user who will try to use your playbook will not get confused.

If no, please justify

SCStelz commented:
Yes, I mean Secret Identifier (URL), I've updated the Readme with screenshots and updated verbiage to reflect this.

From: Manish Kumar ***@***.***>
Sent: Wednesday, January 11, 2023 9:21 PM
To: Azure/Azure-Sentinel ***@***.***>
Cc: Chris Stelzer ***@***.***>; Mention ***@***.***>
Subject: Re: [Azure/Azure-Sentinel] Added new Playbook - Push Defender for Endpoint Alerts including Alert Names, MITRE Tactics, Techniques and Sub-Techniques as Hunting ARM Templates into a Sentinel Github Repository (PR #6803)


@manishkumar1991 commented on this pull request.

________________________________

In Playbooks/Put-MDEAlert-Hunting-GitHub/readme.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2FAzure-Sentinel%2Fpull%2F6803%23discussion_r1067710196&data=05%7C01%7Cchris.stelzer%40microsoft.com%7C86db299ab0f04c4cc73d08daf45ccc4b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638090976674538618%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=lWYFIn9c89OEw6qwDjU9NgVvcmeKjzF9C7JqbxszAVo%3D&reserved=0>:

> +Many of these detections are already covered under Defender for Endpoint, and it would be helpful to easily hunt for certain type of Defender for Endpoint alerts by their associated MITRE tactics, techniques and sub-techniques within the Microsoft Sentinel Hunting interface.

+

+   ![Sentinel](./Media/Sentinel-Hunting.png)

+   ![MDE](./Media/MDE-alert.png)

+

+You can see a sample of a generated JSON ARM Hunting Template for Sentinel here - [MDE Alert JSON ARM Hunting Template](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Playbooks/Put-MDEAlert-Hunting-GitHub/Media/MDE_A_process_was_injected_with_potentially_malicious_code_T1055_T1055_001_T1055_002_T1055_003_T1055_004_T1055_012_T1059_001.json)

+

+## Prerequisites

+

+You will need an existing Microsoft Sentinel workspace, connected to a GitHub repository for [Deploying Custom Content from your repository](https://learn.microsoft.com/en-us/azure/sentinel/ci-cd?tabs=github).

+

+You will need [Defender for Endpoint alerts Syncronized to Sentinel](https://learn.microsoft.com/en-us/azure/sentinel/microsoft-365-defender-sentinel-integration) which populates the required SecurityAlerts table in Sentinel.

+

+You will need to generate a [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for your Github Repository associated with Sentinel with the Repo, and Workflow permissions.

+

+You will need your Personal Access Token from GitHub stored in Keyvault and have the URL of the secret handy.

@SCStelz<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FSCStelz&data=05%7C01%7Cchris.stelzer%40microsoft.com%7C86db299ab0f04c4cc73d08daf45ccc4b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638090976674538618%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=QzR0y5CQN2TIawcrGvfXpwB606w4aunoheblVBrQ78Q%3D&reserved=0> , By URL of secret, do you mean by the URL present inside the secret like mentioned in screen shot below .
[image]<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F97503740%2F211983484-1a448e14-7716-4e0b-b303-309215d6182a.png&data=05%7C01%7Cchris.stelzer%40microsoft.com%7C86db299ab0f04c4cc73d08daf45ccc4b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638090976674538618%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=yr4a22NsJV4IfO0hVozMm46tiI81iqb6VWpq9Pvxp9c%3D&reserved=0>

if Yes , then please mention the steps as well on how to fetch the url of the secret. so that user who will try to use your playbook will not get confused.

If no, please justify

-
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2FAzure-Sentinel%2Fpull%2F6803%23pullrequestreview-1244979573&data=05%7C01%7Cchris.stelzer%40microsoft.com%7C86db299ab0f04c4cc73d08daf45ccc4b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638090976674538618%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=NV7xBXcos5ZXesIB6%2FXOKriYbo1Wad0yDZNjVjTdcl8%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FARQXT63SXJILDGYSRRHV2ZDWR6ID5ANCNFSM6AAAAAASSMAXPE&data=05%7C01%7Cchris.stelzer%40microsoft.com%7C86db299ab0f04c4cc73d08daf45ccc4b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638090976674538618%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=cRc9b9J4TXRbQO0qiFx0%2BIf78oGd6%2BArnuJOmdin3ZI%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

manishkumar1991 commented on Playbooks/Put-MDEAlert-Hunting-GitHub/readme.md:
```diff
@@ -0,0 +1,81 @@
+![Secret](./Media/secret-identifier.png)
+
+This Logic App retrieves the GitHub Personal Access Token from Key Vault using the System Managed Identity and as such you should review and complete the steps in the article [Secure authentication for Conditional Access automation](https://github.com/Azure-Samples/azure-ad-conditional-access-apis/blob/main/00-prereq/readme.md) to create a Key Vault and connect it to a Managed Identity.
+
```
@SCStelz , can't we simply assign the access policies to the keyvault which allows the playbook/logic app to fetch the secret instead of going through the longer procedure mentioned in URL above , 

![image](https://user-images.githubusercontent.com/97503740/212249932-eba3d06f-1798-4adf-9d25-f1a2612e5b51.png)

For me assigning the access policies works 

If you agree with this, please update the readme file with screenshot and steps, or let me know your thoughts.

SCStelz commented:
Yes, that's how it works, I've removed the reference to the article you commented about, I used to use that in my other Playbooks as good reference material. However you're right, an Application Registration is not needed in this Playbook (only access given to the Managed Identity for Keyvault), so that reference doesn't make sense anymore.

From: Manish Kumar ***@***.***>
Sent: Thursday, January 12, 2023 10:13 PM
To: Azure/Azure-Sentinel ***@***.***>
Cc: Chris Stelzer ***@***.***>; Mention ***@***.***>
Subject: Re: [Azure/Azure-Sentinel] Added new Playbook - Push Defender for Endpoint Alerts including Alert Names, MITRE Tactics, Techniques and Sub-Techniques as Hunting ARM Templates into a Sentinel Github Repository (PR #6803)


@manishkumar1991 commented on this pull request.

________________________________

In Playbooks/Put-MDEAlert-Hunting-GitHub/readme.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2FAzure-Sentinel%2Fpull%2F6803%23discussion_r1068967542&data=05%7C01%7Cchris.stelzer%40microsoft.com%7C7a52de2c62be4696597a08daf52d36ab%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638091871798842973%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=zCFknNFN4hUXvAbGKX7Dx82NJ5Bnry9hIZFyqpi10fo%3D&reserved=0>:

> +

+## Prerequisites

+

+You will need an existing Microsoft Sentinel workspace, connected to a GitHub repository for [Deploying Custom Content from your repository](https://learn.microsoft.com/en-us/azure/sentinel/ci-cd?tabs=github).

+

+You will need [Defender for Endpoint alerts Syncronized to Sentinel](https://learn.microsoft.com/en-us/azure/sentinel/microsoft-365-defender-sentinel-integration) which populates the required SecurityAlerts table in Sentinel.

+

+You will need to generate a [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for your Github Repository associated with Sentinel with the Repo, and Workflow permissions.

+

+You will need your Personal Access Token from GitHub stored in Keyvault and have the Secret Identifier (URL) of the secret handy.

+

+You can retrieve your Secret Identifier (URL) by opening the secret and copying the displayed Secret Identifier value as shown in the example below.

+![Secret](./Media/secret-identifier.png)

+

+This Logic App retrieves the GitHub Personal Access Token from Key Vault using the System Managed Identity and as such you should review and complete the steps in the article [Secure authentication for Conditional Access automation](https://github.com/Azure-Samples/azure-ad-conditional-access-apis/blob/main/00-prereq/readme.md) to create a Key Vault and connect it to a Managed Identity.

+

@SCStelz<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FSCStelz&data=05%7C01%7Cchris.stelzer%40microsoft.com%7C7a52de2c62be4696597a08daf52d36ab%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638091871798842973%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Y6zcz3%2FW4RgmCZHC8idb057DU9Y%2FLO69kB7dv6kTe5g%3D&reserved=0> , can't we simply assign the access policies to the keyvault which allows the playbook/logic app to fetch the secret instead of going through the longer procedure mentioned in URL above ,

[image]<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F97503740%2F212249932-eba3d06f-1798-4adf-9d25-f1a2612e5b51.png&data=05%7C01%7Cchris.stelzer%40microsoft.com%7C7a52de2c62be4696597a08daf52d36ab%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638091871798842973%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=SX6Eo1KrRLf7eABwLDGXNjBxHf%2BTRi2600fYHWy2ULI%3D&reserved=0>

For me assigning the access policies works

If you agree with this, please update the readme file with screenshot and steps, or let me know your thoughts.

-
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2FAzure-Sentinel%2Fpull%2F6803%23pullrequestreview-1246952575&data=05%7C01%7Cchris.stelzer%40microsoft.com%7C7a52de2c62be4696597a08daf52d36ab%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638091871798842973%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=BjEKoQHE%2BJKkCFhL1Yg25SySd%2Fe4qJyMV8F9fsyrJXs%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FARQXT6ZVUVBE6UREXG66Z5DWSDW6PANCNFSM6AAAAAASSMAXPE&data=05%7C01%7Cchris.stelzer%40microsoft.com%7C7a52de2c62be4696597a08daf52d36ab%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638091871798842973%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=8Z%2Fpp%2FXrysYiOGViBHt7Vsv9vZ6GXk4LPlnkd6Xpaag%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

manishkumar1991 commented on Playbooks/Put-MDEAlert-Hunting-GitHub/readme.md:
```diff
@@ -0,0 +1,78 @@
+## Step 2: Add your Logic App System Managed Identity to Key Vault with the right permissions
+
+This logic app uses a System Managed Identity to access secrets from Key Vault. As a prerequisite you must have a Key Vault created with your GitHub API Key stored and give this Logic App's System Managed Identity the ability to Read the associated Secret at the Key Vault Git Hub Credentials URL. To learn more about how to use managed identities within Logic Apps, see the article [**Logic Apps an ...
+
```
Steps are looking good, 
But from customer point of view and large audience, I recommend if you can tell them straight on what to do related to key vault access policies, 
Like mentioned below:

A. Assign access policy on key vault for Playbook to fetch the secret key

- Select the Keyvault resource where you have stored the secret
- Click on Access policies Blade
- Click on Create
- Under Secret permissions columun , Select Get , List from "Secret Management Operations"
- Click next to go to Principal tab and choose your deployed playbook name
- Click Next leave application tab as it is.
- Click Review and create
- Click Create

manishkumar1991 replied:
@SCStelz  , Any update on this

manishkumar1991 commented:
@SCStelz ,
As per the Api call used for GitHub in logic app. I sense that Api call will look for hunting content in specified folder/dir. which I think you didn't mention in readme file, that user must a /content/hunting kind of folder structure in github , 

![image](https://user-images.githubusercontent.com/97503740/212649830-cc4703bc-4bbb-490c-ac81-a78c9fe05244.png)


But if you have done it already, and I am able to see it, please redirect me to that, 
If in actually those steps are not there, requesting you to kindly add them in readme file as well. 

Rest everything of fine with PR

SCStelz commented:
Good point, the GitHub API will create the Hunting folder automatically during the API Push of the ARM templates, but it would be a good idea to ensure the Hunting folder exists on the initial run. I've updated the Readme file to indicate this, thanks.

From: Manish Kumar ***@***.***>
Sent: Monday, January 16, 2023 1:57 AM
To: Azure/Azure-Sentinel ***@***.***>
Cc: Chris Stelzer ***@***.***>; Mention ***@***.***>
Subject: Re: [Azure/Azure-Sentinel] Added new Playbook - Push Defender for Endpoint Alerts including Alert Names, MITRE Tactics, Techniques and Sub-Techniques as Hunting ARM Templates into a Sentinel Github Repository (PR #6803)


@SCStelz<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FSCStelz&data=05%7C01%7Cchris.stelzer%40microsoft.com%7Caa8d9790d5704c3223d808daf7a8097c%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638094598333125774%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=o8kkMrBpJhEVUBMfk4fgXtlh8dmScB3wGiXOH8bYRyA%3D&reserved=0> ,
As per the Api call used for GitHub in logic app. I sense that Api call will look for hunting content in specified folder/dir. which I think you didn't mention in readme file, that user must a /content/hunting kind of folder structure in github ,

[image]<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F97503740%2F212649830-cc4703bc-4bbb-490c-ac81-a78c9fe05244.png&data=05%7C01%7Cchris.stelzer%40microsoft.com%7Caa8d9790d5704c3223d808daf7a8097c%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638094598333125774%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=0Hv4Veo9SHaq2ERO2fsaMpG2rO3yOYbPM%2BpFiUgBASk%3D&reserved=0>

But if you have done it already, and I am able to see it, please redirect me to that,
If in actually those steps are not there, requesting you to kindly add them in readme file as well.

Rest everything of fine with PR

-
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2FAzure-Sentinel%2Fpull%2F6803%23issuecomment-1383779450&data=05%7C01%7Cchris.stelzer%40microsoft.com%7Caa8d9790d5704c3223d808daf7a8097c%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638094598333281701%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=kYUhLL9ooH9XErMMA3dmEhPWk49II%2BJe8b5GMG%2BKk6Y%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FARQXT67N4UUHZG363LPMAYDWSULPNANCNFSM6AAAAAASSMAXPE&data=05%7C01%7Cchris.stelzer%40microsoft.com%7Caa8d9790d5704c3223d808daf7a8097c%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638094598333281701%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=71s2eaQRdLUCPod7tkon9pPNKM1DnRuB63EAUQkJDBw%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

manishkumar1991 commented:
@SCStelz Kindly check my comments on readme file as well , regarding key vault access policies

SCStelz commented:
Ohh sorry, apologies, I missed those comments, I've updated the Readme with detailed steps for adding access policy to Keyvault including a screenshot of the policy, thanks.

From: Manish Kumar ***@***.***>
Sent: Monday, January 16, 2023 7:46 PM
To: Azure/Azure-Sentinel ***@***.***>
Cc: Chris Stelzer ***@***.***>; Mention ***@***.***>
Subject: Re: [Azure/Azure-Sentinel] Added new Playbook - Push Defender for Endpoint Alerts including Alert Names, MITRE Tactics, Techniques and Sub-Techniques as Hunting ARM Templates into a Sentinel Github Repository (PR #6803)


@SCStelz<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FSCStelz&data=05%7C01%7Cchris.stelzer%40microsoft.com%7C67eee0d3566d438e413d08daf83d6373%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638095239790159226%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=rVf3HoHMV%2BepCnEsojkuSgVRboIb065gGpl89Qji1j8%3D&reserved=0> Kindly check my comments on readme file as well , regarding key vault access policies

-
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2FAzure-Sentinel%2Fpull%2F6803%23issuecomment-1384796455&data=05%7C01%7Cchris.stelzer%40microsoft.com%7C67eee0d3566d438e413d08daf83d6373%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638095239790315885%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=UxQKO0XvoLuOfEp3XrK1jlW8yGukb6lZ3JSJIim5Wr4%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FARQXT644PR5ZCMGJHWB7DWTWSYIYRANCNFSM6AAAAAASSMAXPE&data=05%7C01%7Cchris.stelzer%40microsoft.com%7C67eee0d3566d438e413d08daf83d6373%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638095239790315885%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=0AdmDwmIAMmfRff6OyqJcUoisAz2FqRNv%2BWU2BCLQVc%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

v-atulyadav commented:
Hi @manishkumar1991, your comments have been responded to. Please review and respond. Thanks

v-atulyadav commented:
Hi @manishkumar1991, @SCStelz has responded to your comments, please check and respond accordingly. Thanks.

manishkumar1991 reviewed (approved):
Approved
